source/api/content-state/0.1/index.md

```diff
@@ -0,0 +1,289 @@
+* The client looks for the content state URL or data as the value of an HTTP POST request parameter `iiif-content`
+* The client allows the content state URL or data to be pasted into part of its UI (e.g., from a "Load..." option). This same mechanism may also accept a Manifest or Collection resource URL directly.
+* The client exposes the current content state in its UI (e.g., "Share this")
+* The client looks for the content state URL or data in an initialisation parameter `data-iiif-content`
+* The data is a parameter in an HTML5 Drag and Drop operation (LINK TO COOKBOOK RECIPE...)
+
+
+## 4. Further Examples
+
+The following examples demonstrate the use of the existing IIIF Presentation API and W3C Web Annotation Data Model to describe parts of resources. The full form of the annotation (as if it were available at the URL given in the `id` property) has been used in each case.  Further examples can be found in the [IIIF Cookbook][annex-cookbook]
+
+### 4.1. A region of a canvas in a manifest
+
+```json
+{
+  "@context": "http://iiif.io/api/presentation/{{ page.major }}/context.json",
+  "id": "https://example.org/import/1",
+  "type": "Annotation",
+  "motivation": ["highlighting"],
+  "target": {
+     "id": "https://example.org/object1/canvas7#xywh=1000,2000,1000,2000",
+     "type": "Canvas",
+     "partOf": [{
+        "id": "https://example.org/object1/manifest",
+        "type": "Manifest"
+     }]
+   }
+}
+```
+
+When processed by a viewer, the user should see the rectangle `1000,2000,1000,2000` highlighted on the Canvas given in the `id` parameter; the viewer loads the manifest linked to in the `partOf` property and navigates to that canvas, and then fills the viewport with that rectangle or otherwise draws attention to it.
+
+
+### 4.2. Start playing at a point in a recording
+
+```json
+{
+  "@context": "http://iiif.io/api/presentation/{{ page.major }}/context.json",
+  "id": "https://example.org/import/2",
+  "type": "Annotation",
+  "motivation": ["highlighting"],
+  "target": {
+    "type": "SpecificResource",
+    "source": {
+        "id": "https://example.org/iiif/id1/canvas1",
+        "type": "Canvas",
+        "partOf": [{
+            "id": "https://example.org/iiif/id1/manifest",
+            "type": "Manifest"
+        }]
+    },
+    "selector": {
+        "type": "PointSelector",
+        "t": 14.5
+	  }
+  }
+}
+```
+
+This example should cause a viewer to open Manifest https://example.org/iiif/id1/manifest, navigate to Canvas https://example.org/iiif/id1/canvas1, and start playing at 14.5 seconds into that canvas.
+
+
+### 4.3. Multiple targets for a comparison view
+
+```json
+{
+  "@context": "http://iiif.io/api/presentation/{{ page.major }}/context.json",
+  "id": "https://example.org/import/3",
+  "type": "Annotation",
+  "motivation": "highlighting",
```

Comment:
["highlighting"]

Reply:
Not necessarily -- the Annotation spec doesn't require the []s in its context.  This is #1764.